macros/models/_sector/dex/dex_sandwiches.sql

```diff
@@ -0,0 +1,55 @@
+{% macro dex_sandwiches(blockchain, transactions) %}
+
+SELECT distinct s1.blockchain
+, s1.project
+, s1.version
+, s1.block_time
+, CAST(date_trunc('month', s1.block_time) AS date) AS block_month
+, tx1.block_number
+, s1.token_sold_address
+, s1.token_bought_address
+, s1.token_sold_symbol
+, s1.token_bought_symbol
+, s1.maker
+, s1.taker
+, s1.tx_hash
+, s1.tx_from
+, s1.tx_to
+, s1.project_contract_address
+, s1.token_pair
+, tx1.index
+, s1.token_sold_amount_raw
+, s1.token_bought_amount_raw
+, s1.token_sold_amount
+, s1.token_bought_amount
+, s1.evt_index
+--, CASE WHEN s1.tx_from=s2.tx_from THEN 'tx_from' ELSE 'taker' END AS commonality
+--, CASE WHEN s1.token_bought_address=s2.token_sold_address THEN 'token_sold' ELSE 'token_bought' END AS sandwiched_token
+--, CASE WHEN s1.token_bought_address<s1.token_sold_address THEN 0 ELSE 1 END AS token_order
+FROM {{ ref('dex_trades') }} s1
+INNER JOIN {{ ref('dex_trades') }} s2 ON s1.blockchain='ethereum'
```

Comment:
Maybe is due to this line?
```suggestion
INNER JOIN {{ ref('dex_trades') }} s2 ON s1.blockchain = '{{blockchain}}'
```

Reply:
Ah yeah ofc 😅 fixed it, thanks!